components/date_picker/DatePicker.js

```diff
@@ -70,40 +71,45 @@ const factory = (Input, DatePickerDialog) => {
       }
     }
 
+    toggleActive = (active) => {
```

Comment:
Since you are passing an argument, I wouldn't call this method `toggleActive` but `setActive` or something like that. `toggleActive` suggest more like changing a boolean to its opposite.